source/administration/changelog.md

```diff
@@ -4,6 +4,130 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+5. A new `config.json` setting to whitelist types of protocols for auto-linking has been added. [Ticket #9547](https://mattermost.atlassian.net/browse/MM-9547).
+6. A new `config.json` setting to disable the [permanent APIv4 delete team parameter](https://api.mattermost.com/#tag/teams%2Fpaths%2F~1teams~1%7Bteam_id%7D%2Fput) has been added. The setting will be off by default for all new and existing installs, except those deployed on GitLab Omnibus. A System Administrator can enable the API v4 endpoint from the config.json file. [Ticket #9916](https://mattermost.atlassian.net/browse/MM-9916).
+7. An unused `ExtraUpdateAt` field has been removed from the channel model. [Ticket #9739](https://mattermost.atlassian.net/browse/MM-9739).
+ 
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under ``"ServiceSettings":`` in ``config.json``:
+   - Added ``"EnableAPITeamDeletion": false,`` to disable the permanent APIv4 delete team parameter.
+   - Added ``"ExperimentalEnableHardenedMode": false`` to
+ - Under ``"EmailSettings":`` in ``config.json``:
+   - Added ``"EnablePreviewModeBanner": true,`` to allow Preview Mode banner to be displayed so users are aware that email notifications are disabled.
+ - Under ``"ClusterSettings":`` in ``config.json``:
+   - Added ``"StreamingPort": 8075,`` to
+   - Added ``"MaxIdleConns": 100,`` to
+   - Added ``"MaxIdleConnsPerHost": 128,`` to
+   - Added ``"IdleConnTimeoutMilliseconds": 90000`` to
+ - Under ``"TeamSettings":`` in ``config.json``:
+   - Added ``"ExperimentalHideTownSquareinLHS": false,`` to
+ - Under ``"DisplaySettings":`` in ``config.json``:
+   - Added ``"CustomUrlSchemes": [],`` to
+ - Under ``"LdapSettings":`` in ``config.json``:
+   - Added ``"LoginIdAttribute": "",`` to add an attribute in the AD/LDAP server used to log in to Mattermost.
+   
+#### API Changes
+
+ - Improved file upload API to stream files instead of loading them entirely into memory.
+ - All APIv3 endpoints were removed.
+ - SAML login endpoints were moved out of API package.
+ - ``context.go`` was moved out of Api4 and into web.
+ - ``api4/handlers.go`` was created to create the API handlers using the Context and Handler from web.
+ - ``web/handlers.go`` was added to define the Handler struct, the base ServeHTTP function and a single web handler.
+ 
+#### WebSocket Changes
+
+ - Ping/pong and reconnection handling were added to Go WebSocket client.
+ - Support was added for WebSocket custom dialer.
+
```

Comment:
"- Added ``channel_converted`` websocket event which is published team-wide whenever a channel is converted from public into private"

(See https://github.com/mattermost/mattermost-server/pull/8798 and https://github.com/mattermost/mattermost-redux/pull/502)